partiql-types/src/main/kotlin/org/partiql/value/PartiQLValue.kt

```diff
@@ -557,6 +557,7 @@ public abstract class MissingValue : PartiQLValue {
 public fun PartiQLValue.toIon(): IonElement = accept(ToIon, Unit)
 
 @PartiQLValueExperimental
+@Throws(TypeCheckException::class)
```

Comment:
Nice catch. Not sure how this was missed!! 🙃

Reply:
Holding off on rebasing to structValue improvements. See https://github.com/partiql/partiql-lang-kotlin/pull/1296/files#diff-b7da874c392ff1e927b2114d2c93f60443b00f42182562e1d2e4c20cf288b0ba which I'm working into main